From 34238c257d3cb6fe7eba247c7e40a1dd49c4f3e8 Mon Sep 17 00:00:00 2001
From: Richard Bradley <richard.bradley@joyent.com>
Date: Thu, 17 Aug 2017 15:59:45 +0000
Subject: [PATCH] MANATEE-352 Add support for "synchronous_commit=off" in
 pg_overrides Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved
 by: Cody Peter Mello <cody.mello@joyent.com>

---
 lib/postgresMgr.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/postgresMgr.js b/lib/postgresMgr.js
index 1985afd..be7f693 100644
--- a/lib/postgresMgr.js
+++ b/lib/postgresMgr.js
@@ -123,7 +123,8 @@ var TUNABLES_SCHEMA = {
                 enum: [
                     'on',
                     'remote_apply',
-                    'remote_write'
+                    'remote_write',
+                    'off'
                 ]
             }
         },
-- 
2.21.0

